pkgs/top-level/all-packages.nix

```diff
@@ -3323,6 +3323,10 @@ with pkgs;
 
   earlyoom = callPackage ../os-specific/linux/earlyoom { };
 
+  easycrypt = callPackage ../applications/science/logic/easycrypt { };
+
+  easycrypt-runtest = callPackage ../applications/science/logic/easycrypt/runtest.nix { };
```

Comment:
Why is that not part of easycrypt?

Reply:
What do you mean?

Reply:
Would you like to elaborate or can I merge as is?